### PR TITLE
Adding ansible deploy for OSCAP e2e

### PR DESCRIPTION
**Pull request by san7ket**

### CODE REVIEW

**Comment by san7ket** [comment]
All tests are skipped due to two BZ's https://bugzilla.redhat.com/show_bug.cgi?id=1716307 and https://bugzilla.redhat.com/show_bug.cgi?id=1722475

**Comment by san7ket** [comment]
I have tested the tests for new added ansible test case with workaround /Satellite 6/job/satellite6-standalone-automation/1585/ It passes.
For other tests there, is no workaround to test it , you can see the failure is due to BZ https://bugzilla.redhat.com/show_bug.cgi?id=1722475
/view/Satellite 6/job/satellite6-standalone-automation/1590/

**Comment by oshtaier** [comment]
do you feel that working with static ids (e.g. id=1) is safe?

**Comment by san7ket** [comment]
Will update those

**Comment by san7ket** [comment]
Comments resolved

**Comment by san7ket** [comment]
@ntkathole @jameerpathan111 Please review

File: tests/foreman/longrun/test_oscap.py [hunk 1]
```diff
@@ -526,6 +551,145 @@ def test_positive_oscap_run_with_tailoring_file_and_capsule(self):
...
+                'name': vm.hostname.lower(),
+                'lifecycle-environment': self.config_env['env_name'],
+                'content-view': self.config_env['cv_name'],
+                'hostgroup': vm_values.get('hgrp'),
+                'openscap-proxy-id': self.proxy_id,
+                'organization': self.config_env['org_name'],
+                'ansible-role-ids': role_id
+            })
+            # needed to work around BZ#1656480
+            if bz_bug_is_open(1656480):
```
Comment (ntkathole) [inline tests/foreman/longrun/test_oscap.py]: Bugzilla is closed as duplicate of https://bugzilla.redhat.com/show_bug.cgi?id=1650103

File: tests/foreman/longrun/test_oscap.py [hunk 2]
```diff
@@ -159,28 +169,31 @@ def configure_puppet_test(cls):
...
+        """Perform end to end oscap test and upload reports via puppet
 
         :id: 17a0978d-64f9-44ad-8303-1f54ada08602
 
         :expectedresults: Oscap reports from rhel6 and rhel7 clients should be
             uploaded to satellite6 and be searchable.
 
         :CaseLevel: System
+
+        :BZ: 1479413
```
Comment (ntkathole) [inline tests/foreman/longrun/test_oscap.py]: s/1479413/1722475

File: tests/foreman/longrun/test_oscap.py [hunk 3]
```diff
@@ -297,6 +315,8 @@ def test_positive_push_updated_content(self):
             satellite should get updated reports
 
         :CaseLevel: System
+
+        :BZ: 1420439
```
Comment (ntkathole) [inline tests/foreman/longrun/test_oscap.py]: s/1479413/1722475

File: tests/foreman/longrun/test_oscap.py [hunk 4]
```diff
@@ -353,15 +375,14 @@ def test_positive_push_updated_content(self):
...
             Host.update({
                 'name': vm.hostname.lower(),
                 'lifecycle-environment': self.config_env['env_name'],
                 'content-view': self.config_env['cv_name'],
                 'hostgroup': vm_values.get('hgrp'),
-                'openscap-proxy-id': 1,
+                'openscap-proxy-id': self.proxy_id,
                 'organization': self.config_env['org_name'],
-                'environment': 'production'
+                'environment-id': self.proxy_id,
```
Comment (ntkathole) [inline tests/foreman/longrun/test_oscap.py]: proxy id is assigned to environment id?

**Comment by oshtaier** [comment]
ACK =)

File: tests/foreman/longrun/test_oscap.py (hunk 3 above)
Comment (san7ket) [reply tests/foreman/longrun/test_oscap.py]: will add the other BZ aswell , the one mentioned is valid aswell

File: tests/foreman/longrun/test_oscap.py (hunk 1 above)
Comment (san7ket) [reply tests/foreman/longrun/test_oscap.py]: ahh forgot

**Pull request merged**